.github/workflows/continuous_production.yml

```diff
@@ -0,0 +1,53 @@
+name: Continuous production
+on:
+  - push
+  # - pull_request # already covered by push
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+          if [ -f Gopkg.toml ]; then
+              curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+              dep ensure
+          fi
+      - name: Build
+        run: go build -v .
+      - name: Run tests
+        run: go test ./...
+  deploy:
+    name: Serverless deploy master branch
+    runs-on: ubuntu-latest
+    needs: build
+    # Uncomment after the build is green on this PR
```

Comment:
Did we want to un-comment this now?

Reply:
If I uncomment this now, the deployment will not be tested anymore in this PR request if further changes need to be made (cc @mourtisma), so I must do this either before merging this PR once it is approved (or via a dirty commit in master afterwards in case I forget)

Reply:
So I guess this means you approve this PR without this line ?

Reply:
Got it. Then I approve this PR. let's get @mourtisma 's review and then you can un-comment it and merge this